src/status_im/ui/screens/usage_data/views.cljs

```diff
@@ -28,12 +28,9 @@
      [react/view styles/bottom-button-container
       [components.common/button {:button-style      styles/share-button
                                  :uppercase?        false
-                                 :on-press          #(utils/show-confirmation {:ios-confirm-style "default"}
-                                                                              (i18n/label :t/confirmation-title)
-                                                                              (i18n/label :t/confirmation-text)
-                                                                              (i18n/label :t/confirmation-action)
-                                                                              (fn [] (re-frame/dispatch [:help-improve-handler true next]))
-                                                                              nil)
+                                 ;; this is a desktop hack. we are currently not collecting data usage
+                                 ;; on behalf of the user
+                                 :on-press          #(re-frame/dispatch [:help-improve-handler false next])
```

Comment:
Should we outline that we do this only for `desktop` platform?

Reply:
Agreee. Every place where we modify few lines better to be enclosed with `(if platform/desktop? ..). That will make merging with `defelop` easier.

Reply:
right. agreed with you both. thanks for the feedback. will make the follow up change.